main/src/library/Assert.flix

```diff
@@ -157,6 +157,107 @@ mod Assert {
             }
         }
 
+    ///
+    /// Handles the `Assert` effect by ignoring all assertions (no-op).
+    ///
+    pub def handleWithNoOp(f: a -> b \ ef): (a -> b \ (ef - Assert)) = x ->
+        run {
+            f(x)
+        } with handler Assert {
+            def assert(_cond, _msg, resume) = resume(())
+        }
+
+    ///
+    /// Handles the `Assert` effect by logging assertion failures using the `Logger` effect.
+    ///
+    /// Assertions continue execution after logging the assertion failure.
+    ///
+    pub def handleWithLogger(f: a -> b \ ef): (a -> b \ (ef - Assert) + Logger) = x ->
+        run {
+            f(x)
+        } with handler Assert {
+            def assert(cond, msg, resume) = {
+                if (cond)
+                    resume(())
+                else {
+                    Logger.warn(msg);
+                    resume(())
+                }
+            }
+        }
+
+    ///
+    /// Handles the `Assert` effect by printing assertion failures to stdout.
+    ///
+    /// Assertions continue execution after printing the assertion failure.
+    ///
+    pub def handleWithStdOut(f: a -> b \ ef): (a -> b \ (ef - Assert) + IO + Sys) = x ->
+        run {
+            f(x)
+        } with handler Assert {
+            def assert(cond, msg, resume) = {
+                if (cond)
+                    resume(())
+                else {
+                    println(RichString.toAnsiString(msg));
+                    resume(())
+                }
+            }
+        }
+
+    ///
+    /// Handles the `Assert` effect by printing assertion failures to stderr.
+    ///
+    /// Assertions continue execution after printing the assertion failure.
+    ///
+    pub def handleWithStdErr(f: a -> b \ ef): (a -> b \ (ef - Assert) + IO + Sys) = x ->
+        run {
+            f(x)
+        } with handler Assert {
+            def assert(cond, msg, resume) = {
+                if (cond)
+                    resume(())
+                else {
+                    use Console.eprintln;
+                    run {
+                        eprintln(RichString.toAnsiString(msg))
```

Comment:
This is because we have no eprintln in prelude. But perhaps we should have that instead?